File: user-api/go-sdk/pkg/models/operations/getuserloginparametersv2.go
package operations

import (
	"openapi/pkg/models/shared"
)

type GetUserLoginParametersV2PathParams struct {
	Username string `pathParam:"style=simple,explode=false,name=username"`
}

type GetUserLoginParametersV2200ApplicationJSON struct {
	LoginParameters []shared.LoginParameters `json:"login_parameters,omitempty"`
}

type GetUserLoginParametersV2Request struct {
	PathParams GetUserLoginParametersV2PathParams
}

type GetUserLoginParametersV2Response struct {
	ContentType                                      string
	StatusCode                                       int64
	GetUserLoginParametersV2200ApplicationJSONObject *GetUserLoginParametersV2200ApplicationJSON
}

File: user-api/ts-sdk/src/sdk/models/operations/getuserloginparametersv2.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class GetUserLoginParametersV2PathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=username" })
  username: string;
}


export class GetUserLoginParametersV2200ApplicationJson extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=login_parameters", elemType: shared.LoginParameters })
  loginParameters?: shared.LoginParameters[];
}


export class GetUserLoginParametersV2Request extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: GetUserLoginParametersV2PathParams;
}


export class GetUserLoginParametersV2Response extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  getUserLoginParametersV2200ApplicationJSONObject?: GetUserLoginParametersV2200ApplicationJson;
}
